reports/manuscript/abstract.tex

```diff
@@ -13,15 +15,15 @@ \section*{Abstract}
 lasso (SGL) \cite{simon2013sparse} that takes into account tissue
 properties measured along all of the tracts, and selects informative
 features by enforcing sparsity, not only at the level of individual
-tracts and tissue properties, but also across the entire set of tracts
-and all of the measured tissue properties. The sparsity penalties for
-each of these constraints is identified using a nested cross-validation
-scheme that guards against over-fitting and simultaneously identifies
-the correct level of sparsity. We demonstrate the accuracy of the method
-in two settings: in a regression setting, dMRI is used to accurately
-predict ``brain age'' \cite{yeatman2014lifespan, Brown2012-so}. In this
-case, the weights are distributed throughout the white matter indicating
-that many different regions of the white matter change with development
+tracts, but also across the entire set of tracts and all of the measured
+tissue properties. The sparsity penalties for each of these constraints
+is identified using a nested cross-validation scheme that guards
+against over-fitting and simultaneously identifies the correct level of
+sparsity. We demonstrate the accuracy of the method in two settings:
+in a regression setting, dMRI is used to accurately predict ``brain
+age'' \cite{yeatman2014lifespan, Brown2012-so}. In this case, the
+weights are distributed throughout the white matter indicating that
+many different regions of the white matter change with development
```

Comment:
As per Jason's comment on slack. We might want to reverse the order of regression and classification here as well.

Reply:
Thanks! Just pushed that to this PR. Also changed it in the methods section.

Reply:
Yeah - looks great.